android/app/src/main/java/com/microsoft/codepush/react/CodePushDialog.java

```diff
@@ -11,30 +11,23 @@
 
 public class CodePushDialog extends ReactContextBaseJavaModule{
 
-    private Activity mainActivity;
+    private CodePushNativeModule codePushNativeModule;
```

Comment:
Should we start using the "m" prefix for private class members? That seems to be the equivalent of the _ in the Java world, and its what RN uses.